doc/source/whatsnew/v0.22.0.txt

```diff
@@ -109,7 +109,7 @@ I/O
 ^^^
 
 - :func:`read_html` now rewinds seekable IO objects after parse failure, before attempting to parse with a new parser. If a parser errors and the object is non-seekable, an informative error is raised suggesting the use of a different parser (:issue:`17975`)
```

Comment:
can you add a small example in the docs in io.rst as well.

Reply:
Done https://github.com/pandas-dev/pandas/pull/18155/commits/21c5f5ea3528e451ec2653e19de094d25cdc0405